bindings/swift/Sources/OpenDAL/Operator.swift

```diff
@@ -0,0 +1,93 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import COpenDAL
+
+public enum OperatorError: Error {
+    case failedToBuild
+    case operationError(UInt32)
+}
+
+/// A type used to access almost all OpenDAL APIs.
+public class Operator<S: Service> {
+    var nativeOp: opendal_operator_ptr
+    
+    deinit {
+        withUnsafePointer(to: &nativeOp) { nativeOpPtr in
+            opendal_operator_free(nativeOpPtr)
+        }
+    }
+    
+    /// Creates an operator with the given options.
+    ///
+    /// - Parameter options: The option map for creating the operator.
+    /// - Throws: `OperatorError` value that indicates an error if failed.
+    public init(options: [String : String]) throws {
+        var nativeOptions = opendal_operator_options_new()
+        nativeOp = withUnsafeMutablePointer(to: &nativeOptions) { nativeOptionsPtr in
+            defer {
+                opendal_operator_options_free(nativeOptionsPtr)
+            }
+            
+            for option in options {
+                opendal_operator_options_set(nativeOptionsPtr, option.key, option.value)
+            }
+            
+            return opendal_operator_new(S.scheme, nativeOptionsPtr)
+        }
+        
+        guard nativeOp.ptr != nil else {
+            throw OperatorError.failedToBuild
+        }
+    }
+    
+    /// Blockingly write the data to a given path.
+    ///
+    /// - Parameter data: The content to be written.
+    /// - Parameter path: The destination path for writing the data.
+    /// - Throws: `OperatorError` value that indicates an error if failed to write.
+    public func blockingWrite<D: COWData>(_ data: D, to path: String) throws {
```

Comment:
I prefer to start with simple case. Does swift have native bytes or string type? We can accept it here instead.

Reply:
Yes, Swift has `Data` type which is basically a `Vec<u8>`, so we can use it here.

Reply:
Yes, please remove the generic parameter. OpenDAL won't expose that at user's public API.